index.js

```diff
@@ -1,59 +1,96 @@
-const shelve = document.querySelector('#book-shelve');
-const form = document.querySelector('form');
-const bookInput = document.querySelector('#book-name');
-const authorInput = document.querySelector('#author-name');
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
 
-const bookShelve = [];
+const forLs = [];
 
-function storeShelve() {
-  localStorage.setItem('books', JSON.stringify(bookShelve));
-}
+/* eslint max-classes-per-file: ["error", 2] */
 
-function createBook(book, author) {
-  const newBook = {
-    bookName: book,
-    author,
-  };
-  bookShelve.push(newBook);
-
-  const mybook = document.createElement('div');
-  bookShelve.forEach((book, idx) => {
-    mybook.innerHTML = `<div class="book">
-      <h3>${book.bookName}</h3>
-      <p>${book.author}</p>
-      <button class="delete" class="delete">Remove</button>
-      </div>
-      <hr>`;
-
-    const deleteBtn = mybook.querySelector('.delete');
-    deleteBtn.addEventListener('click', () => {
-      bookShelve.splice(idx, 1);
-      mybook.remove();
-      storeShelve();
-    });
-  });
+class DisplayBook {
+  static addBook(newBook, index) {
+    const library = document.querySelector('#book-shelve');
+    if (!localStorage.getItem('books')) {
+      const noBook = document.createElement('p');
+      noBook.innerHTML = 'No books in library';
+      library.appendChild(noBook);
+    }
+    const container = document.createElement('div');
+    container.classList.add('book');
+    container.classList.add('flex');
+    container.innerHTML = `
+    <div class="book-details">
+     <h3>${newBook.title}</h3>
+     <p>${newBook.author}</p>
+    </div>
+     <button class="delete" data-remove=${index}>Delete</button>
```

Comment:
- [ ] Please add the functionality of removing a book as a method to the `BookCollection` class as mentioned in the project requirements.

![image](https://user-images.githubusercontent.com/68796232/182392242-2d04ade4-5019-40c2-a86f-4ff9a8812952.png)

Reply:
pls, I don't get you, you are talking about a function but you pointed out my paragraph.

Actually, I have a delete function there called deleteBook